.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,11 @@
 - **Etcher version:** 
 - **Operating system and architecture:** 
 - **Image flashed:** 
+- **What do you think should have happened:** <!-- even step by step reproduction process -->
```

Comment:
It's not clear what the "even" means here - perhaps it should be removed? Or reworded to something like "or a step by step reproduction process" ?